src/api/controllers/adminPermissionsController.js

```diff
@@ -207,4 +211,155 @@ export default class adminPermissions {
       }
     });
   }
+
+  /**
+   * fetch all reported Articles
+   *
+   * @static
+   * @param {array} req - request array
+   * @param {array} res - respond array
+   * @returns {array} response body
+   * @memberof getArticles
+   */
+  static async getReportedArticles(req, res) {
+    const defaultOffset = 0;
+    const defaultLimit = 10;
+    const offset = req.query.offset || defaultOffset;
+    const limit = req.query.limit || defaultLimit;
+    const paginatedArticle = await Report.findAll({
+      attributes: ['createdAt'],
+      include: [
+        {
+          model: Article,
+          required: true,
+          attributes: ['title', 'slug', 'description', 'body', 'coverImage']
+        },
+        {
+          model: User,
+          required: true,
+          attributes: ['username', 'email']
+        },
+        {
+          model: Reason,
+          required: true,
+          attributes: ['description']
+        }
+      ],
+      offset,
+      limit
+    });
+    res.status(status.OK).send({
+      status: status.OK,
+      paginationDetails: generatePagination(paginatedArticle.length,
+        paginatedArticle,
+        offset,
+        limit),
+      data: paginatedArticle
+    });
+  }
+
+  /**
+   * fetch single reported Article
+   *
+   * @static
+   * @param {array} req - request array
+   * @param {array} res - respond array
+   * @returns {array} response body
+   * @memberof getArticles
+   */
+  static async getReportedArticle(req, res) {
+    const { slug } = req.params;
+    const reportedArticle = await Report.findOne({
+      attributes: [['createdAt', 'ReportedOn']],
```

Comment:
@Ramadhan0 I don't think there's any need to fetch `createdAt` as `reportedOn`. Check for other instances of `reportedOn` and remove, both in your code and tests. Thanks.